src/plugins/intel_gpu/tests/fusions/deconvolution_fusion_test.cpp

```diff
@@ -541,8 +541,8 @@ TEST_P(deconv_scale_actv_quant_i8, basic) {
         data("out_lo", get_mem(get_single_element_layout(p), -127)),
         data("out_hi", get_mem(get_single_element_layout(p), 127)),
         deconvolution("deconv", "input", { "weights" }, p.groups, p.stride, p.pad),
-        scale("scale", "deconv", "scale_data"),
-        activation("actv", "scale", activation_func::softsign),
+        eltwise("scale", { "deconv", "scale_data" }, eltwise_mode::prod),
+        activation("actv", "scale", activation_func::hyperbolic_tan),
```

Comment:
could you explain why you changed softsign to hyperbolic_tan?

Reply:
Test fails in dg2 because softsign activation not supported in onednn.

Reply:
In that case, could you review other approaches as well?
choice 1) Do not fuse such "unsupported post-ops" if the target HW is using onednn: fusion such fusion from `prepare_primitive_fusing`
choice 2) Fallback to cldnn for such case: it should be handled from `get_preferred_impl`
choice 3) just avoid such case as you did and file a ticket in backlog.

Reply:
Option1 requires testcase modification because that makes fuse_count different between cldnn and onednn. Thus I handled it by option2.

Reply:
Current implementation is Option1 (do not fuse activation)